File: solutions/beecrowd/1008/1008.cpp
#include <cstdint>
#include <cstdio>

int main() {
    std::int16_t a, b;
    float c;

    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%f", &c);

    printf("NUMBER = %d\n", a);
    printf("SALARY = U$ %.2f\n", c * b);

    return 0;
}

File: solutions/beecrowd/1008/1008.go
package main

import "fmt"

func main() {
	var a int16
	var b, c float64

	fmt.Scanf("%d", &a)
	fmt.Scanf("%f", &b)
	fmt.Scanf("%f", &c)

	fmt.Printf("NUMBER = %d\n", a)
	fmt.Printf("SALARY = U$ %.2f\n", b*c)
}
